test_runner/README.md

```diff
@@ -71,7 +71,8 @@ a subdirectory for each version with naming convention `v{PG_VERSION}/`.
 Inside that dir, a `bin/postgres` binary should be present.
 `DEFAULT_PG_VERSION`: The version of Postgres to use,
 This is used to construct full path to the postgres binaries.
-Format is 2-digit major version nubmer, i.e. `DEFAULT_PG_VERSION="14"`
+Format is 2-digit major version nubmer, i.e. `DEFAULT_PG_VERSION="14"`. Alternatively,
```

Comment:
In docker builds "v14" and "v15" are used. What do you think about having the same logic?

Reply:
Ah, I always get confused where to use what 😄 
Added support for v-prefixed strings like v14/v15 as well